calico-enterprise/threat/web-application-firewall.mdx

```diff
@@ -100,7 +100,7 @@ kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/microserv
 - [Enable WAF on your cluster](#enable-waf-on-your-cluster)
 - [Apply WAF to your services](#apply-waf)
 - [View WAF events](#view-waf-events)
-- [Create global alerts](#create-global-alerts)
+- [Disable WAF feature from your cluster](#disable-waf-in-your-cluster)
```

Comment:
This `#disable-waf-in-your-cluster` anchor link might not exist (and other versioned or CC files). I think it need to be this (or change line 278):

```
[Disable WAF for a service](#disable-waf-for-a-service)
```

Reply:
Fixed link.

Reply:
Fixed.